composer.json

```diff
@@ -26,48 +16,15 @@
     "drupal/pathauto": "^1.3",
     "drupal/radix": "^4.3"
   },
-  "require-dev": {
-    "drush/drush": "~9",
-    "drupal/coder": "~8"
-  },
   "config": {
-    "sort-packages": true,
-    "preferred-install": {
-      "drupal/core": "dist"
-    }
-  },
-  "scripts": {
-    "post-install-cmd": [
-      "@composer drupal-scaffold"
-    ],
-    "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
+    "sort-packages": true
   },
   "extra": {
-    "patchLevel": {
-      "drupal/core": "-p2"
-    },
+    "enable-patching": true,
```

Comment:
Is this also needed in this project? I think it's only read in the main composer project to enable patching from subprojects.

Reply:
Thank you @arlina-espinoza!  😄 I pushed an update to remove it.